flang/lib/Semantics/mod-file.cpp

```diff
@@ -759,6 +759,15 @@ void ModFileWriter::PutObjectEntity(
     PutLower(os << "attributes(", common::EnumToString(*attr))
         << ") " << symbol.name() << '\n';
   }
+  if (symbol.test(Fortran::semantics::Symbol::Flag::CrayPointer)) {
```

Comment:
Please add a LIT test with `test_modfile.py` in `flang/test/Semantics/`.

Reply:
Thanks for the review. I will do that.